pkgs/development/ocaml-modules/cstruct/lwt.nix

```diff
@@ -4,11 +4,11 @@ if lib.versionOlder (cstruct.version or "1") "3"
 then cstruct
 else
 
-buildDunePackage {
-  pname = "cstruct-lwt";
-  inherit (cstruct) version src useDune2 meta;
+  buildDunePackage {
+    pname = "cstruct-lwt";
+    inherit (cstruct) version src meta;
```

Comment:
`minimalOCamlVersion` could be inherited too.

Reply:
Sounds like a bad suggestion…

Reply:
Why?

Reply:
`minimalOCamlVersion` is consumed by `buildDunePackage` and is not an attribute to the resulting derivation.

So you cannot write `inherit (cstruct) minimalOCamlVersion` (well, this can be written but won’t evaluate).

Moreover, by design, this information is implicitly inherited. It’s not needed to repeat the information that’s already there.

Reply:
Thanks.

Reply:
> Moreover, by design, this information is implicitly inherited. It’s not needed to repeat the information that’s already there.

So, just remove it then?

Reply:
What do you mean?